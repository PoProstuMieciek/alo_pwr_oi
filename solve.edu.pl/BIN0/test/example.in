3
7
10
16