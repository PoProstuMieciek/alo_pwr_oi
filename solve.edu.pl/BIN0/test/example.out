111
1010
10000
